snowexsql/tables/point_data.py

```diff
@@ -33,7 +33,6 @@ class PointData(SingleLocationData, Measurement, Base):
     value = Column(Float)
 
     # bring these in instead of Measurement
-    type = Column(String())
     units = Column(String())
 
     # Link the instrument id with a foreign key
```

Comment:
Can we sneak in the same inheritance for the `instrument` as you did with DOI and MeasurementType?

Reply:
I believe so!